app/models/build.rb

```diff
@@ -2,7 +2,7 @@
 class Build < ActiveRecord::Base
   SHA1_REGEX = /\A[0-9a-f]{40}\Z/i
   SHA256_REGEX = /\A(sha256:)?[0-9a-f]{64}\Z/i
-  DIGEST_REGEX = /\A[\w._-]+\/[\w\/_-]+@sha256:[0-9a-f]{64}\Z/i
+  DIGEST_REGEX = /\A[\w\.-]+\/[\w\/-]+@sha256:[0-9a-f]{64}\Z/i
```

Comment:
ah yeah ... must have been the _ that triggered the warning I saw ... `.` does not cause a warning ... removed ... even though I think `\.` might be less ambiguous